tests/Helpers/FixtureHelpers.php

```
@@ -0,0 +1,46 @@
+use App\Models\Project;
+use App\Models\TaskStatus;
+use App\Models\User;
+
+trait FixtureHelpers
+{
+    private function createMeeting(User $user, Project $project): void
+    {
+        Meeting::factory()->for($user)->for($project)->create();
+    }
+
+    private function createApiToken(User $user, ?Project $project = null): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$project'. (undefined)

(UnusedFormalParameter)